client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -1140,6 +1141,19 @@ static Request xpackUsage(XPackUsageRequest usageRequest) {
         return request;
     }
 
+    static Request setIndexLifecyclePolicy(SetIndexLifecyclePolicyRequest setPolicyRequest) {
+        Request request = new Request(HttpPut.METHOD_NAME,
+            new EndpointBuilder()
+                .addCommaSeparatedPathParts(setPolicyRequest.indices())
```

Comment:
Pls be sure this is not null. Other converters do a null check and return and give this `addCommaSeparatedPathParts` a empty array if need be. check `forceMerge` for an example